src/util/ld_util.py

```diff
@@ -189,7 +245,6 @@ def get_skosxl_label_triples_for_skos_concept_from_rdf_store(
     :param sparql_endpoint: the location of the RDF store.
     """
     query_construct_skos_xl_labels = (
-        f"PREFIX skosxl: <http://www.w3.org/2008/05/skos-xl#> "
```

Comment:
moet de PREFIX er uit?

Reply:
van mij niet ;)
Dit is een fout, bedankt voor het opsporen! Ik heb hem teruggezet